src/components/UI/atoms/Icon/Icon.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import * as icomoon from 'styled-icons/icomoon';
+import { iconSizes, icons } from '<variables>/icons';
+import { iconColors } from '<variables>/colorPalette';
+import spacing from '<variables>/spacing';
+
+
+/**
+ * @description - Icon Component
+ *
+ * @prop {string} color - color
+ * @prop {string} height - height
+ * @prop {string} width - width
+ * @prop {string} display - display
+ * @prop {string} margin - margin
+ * @prop {string} icon - icon
+ *
+ * @return {component} Icon
+ */
+const Icon = ({
+  id,
+  color,
+  height,
+  width,
+  display,
+  margin,
+  icon,
+}) => {
+  const IconType = icomoon[icons[icon]];
```

Comment:
what is `icomoon`?

Reply:
icomoon is the name of the icon pack from styled-icon component library.

Reply:
please rename since it is a default export for readability

Reply:
ok